pydata_center/monitoring/tests/load/locustfile.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+import logging
+import os
+import random
+import uuid
+
+from dotenv import load_dotenv
+from locust import HttpUser, between, task
+
+load_dotenv()
+
+logging.basicConfig(
+    level=logging.INFO, format='%(asctime)s %(levelname)s %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+TOKEN = None
+
+USERNAME = os.getenv('LOCUST_USERNAME', 'user')
+PASSWORD = os.getenv('LOCUST_PASSWORD', 'password')
+
+logger.info(f'USERNAME: {USERNAME}')
+logger.info(f'PASSWORD: {PASSWORD}')
+
+
+def get_token_once():
+    """Get authentication token from API.
+    Returns cached token if available."""
+    global TOKEN
+    if TOKEN:
+        return TOKEN
+    import requests
+    response = requests.post('http://127.0.0.1:8000/api/v1/token/', json={
+        'username': USERNAME,
+        'password': PASSWORD
+    })
+    if response.status_code == 200:
+        TOKEN = response.json()['access']
+        logger.info(f'TOKEN: {TOKEN[:20]}...')
+        return TOKEN
+    else:
+        logger.error(f'Login failed: {response.status_code}')
+        return None
+
```

Comment:
Avoid Global TOKEN in Concurrent Testing
Global caching breaks if multiple users in parallel need to re-authenticate (e.g., with multiple usernames). Each simulated user should ideally have isolated tokens.

Suggested Fix:
Make get_token_once an instance method and remove global TOKEN.